services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitFormNew.tsx

```diff
@@ -31,7 +31,6 @@ import {
   maxLength,
   number,
   required,
-  validateSelectOptions,
 } from "@common/utils/Validate";
```

Comment:
validateSelectOptions made app crash with too many updates so I made the decision to _get rid of it_ in this file. ...Fix the component instead? 😄